OutOfSchool/OutOfSchool.WebApi/Models/ApplicationDTO.cs

```diff
@@ -12,7 +12,7 @@ public class ApplicationDto
 {
     public Guid Id { get; set; }
 
-    [JsonConverter(typeof(StringEnumConverter))]
+    [EnumDataType(typeof(ApplicationStatus), ErrorMessage = "Status should be in enum range")]
```

Comment:
are these mutually exclusive? Because FE right now expects enums as string and most enums are sent as string